docs/Getting-Started/Create-bootable-media/bootable-usb-using-windows.md

```diff
@@ -1 +1,93 @@
-Users creating installation media from Windows can use Rufus. Rufus is an open source startup disk creator capable of writing ISO images to portable storage media.
\ No newline at end of file
+# Create a Bootable Pop!\_OS USB in Windows 10
+
+This section explains the process of creating a bootable Pop!_OS USB in macOS using balena Etcher.
+
+- [Install ISO flashing software.](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#install-etcher)
+- [Download the Pop!\_OS.iso file.](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#download-the-pop_osiso-file)
+- [Verify the Pop!\_OS.iso Hash Value](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#verify-the-hash-value)
+- [Flash the ISO to your USB storage device.](/Getting-Started/Create-bootable-media/bootable-usb-using-windows.html#create-the-bootable-usb)
+
+---
+
+## Install Etcher
+
+1. Navigate to Etcher's [download page](https://www.balena.io/etcher/) and click `Download for Windows (x86|x64)`.
+
+![Download Etcher](/images/create-bootable-usb-win10/download_etcher.png)
+
+2. Navigate to the downloaded executable and run it.
+
+![Run Etcher Executable](/images/create-bootable-usb-win10/run-etcher-exe.png)
+
+3. Click `I Agree` and wait for the installation to complete.
+
+![Install Etcher](/images/create-bootable-usb-win10/install-etcher.png)
+
+4. Wait for the installation process to complete.
+
+![Complete Installation](/images/create-bootable-usb-win10/complete-install.png)
+
+## Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](/Getting-Started/Create-bootable-media/create-bootable-usb.html#choose-a-pop_os-image) for information about each version.
+
+Take note of the hash value for your chosen ISO. Use this value to verify the integrity of the ISO file.
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+## Verify the Hash Value
+
+1. Launch the Windows Command Prompt.
+
+![Launch CMD](/images/create-bootable-usb-win10/launch-cmd.png)
+
+2. Navigate to the folder where the Pop!_OS.iso file is located. The user's Downloads folder is used in the below example. Substitute your username for `USERNAME`.
+
+```bash
+cd C:\Users\USERNAME\Downloads
+```
+
+3. Use the `CertUtil` program to verify your ISO file. Make sure the name of the ISO matches exactly what is in your folder. The filename changes depending on the ISO version you chose.
+
+```bash 
+CertUtil -hashfile pop-os_21.10_amd64_intel_8.iso sha256
+```
+
+4. This command will output a long string of characters. Compare the output to the appropriate hash value on the Pop!\_OS download site.
+
```

Comment:
For people with little to no CLI experince a link to a site like https://emn178.github.io/online-tools/sha256_checksum.html might be helpful to give a GUI way to do this.

Reply:
It seems like a waste of bandwidth (and a privacy risk) to upload the iso to a third-party website just to check the hash. We're providing step-by-step instructions, I do not think this is necessary.

Reply:
I see your point, and since I can't think of another GUI way that would be the same for both Mac and Windows, without installing another third-party software, I suppose it is best left as is.